x/trigger/types/msgs.go

```diff
@@ -0,0 +1,163 @@
+package types
+
+import (
+	fmt "fmt"
+
+	codectypes "github.com/cosmos/cosmos-sdk/codec/types"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	sdktx "github.com/cosmos/cosmos-sdk/types/tx"
+)
+
+var _ sdk.Msg = &MsgCreateTriggerRequest{}
+var _ sdk.Msg = &MsgDestroyTriggerRequest{}
+var _ codectypes.UnpackInterfacesMessage = (*MsgCreateTriggerRequest)(nil)
+
+// NewCreateTriggerRequest Creates a new trigger create request
+func NewCreateTriggerRequest(authorities []string, event TriggerEventI, msgs []sdk.Msg) *MsgCreateTriggerRequest {
+	actions, err := sdktx.SetMsgs(msgs)
+	if err != nil {
+		fmt.Printf("unable to set messages: %s\n", err)
```

Comment:
Don't use `Printf` anywhere.

For `cli` stuff, if you have to print something, `Fprintf` should be used using the stdout or stderr from the `cmd`.

For the rest of the stuff, the only printing to that should be done is to logs. Even in unit tests, you should be using `t.Logf` instead.

In the cases in here, this should be returning the error when it's encountered.

Reply:
I guess there's some valid uses of it in the sims. But other than that, it shouldn't be used.

Reply:
I pushed up a fix for this. Feel free to resolve whenever.

Reply:
Thanks, this was left in from testing.